frontend/src/components/VisitRouter/VisitForm.js

```diff
@@ -0,0 +1,98 @@
+import React from "react"
+import PropTypes from "prop-types"
+import Grid from "@material-ui/core/Grid"
+import Select from "@material-ui/core/Select"
+import MenuItem from "@material-ui/core/MenuItem"
+import TextField from "@material-ui/core/TextField"
+import InputLabel from "@material-ui/core/InputLabel"
+import FormControl from "@material-ui/core/FormControl"
+import { PrevPointHeading } from "../Typography"
+import { URGENCY_OPTIONS } from "../../constants"
+
+function VisitForm(props) {
```

Comment:
Any reason we aren't using const and arrow functions for these?
`const VisitForm = (props) => {...}`

Reply:
My personal preference seeping in, Ill switch them back to stay consistent